5e-SRD-Monsters.json

```diff
@@ -420,8 +428,13 @@
         }
       },
       {
-        "name": "Acid Breath (Recharge 5-6)",
+        "name": "Acid Breath",
         "desc": "The dragon exhales acid in a 60-foot line that is 5 feet wide. Each creature in that line must make a DC 18 Dexterity saving throw, taking 54 (12d8) acid damage on a failed save, or half as much damage on a successful one.",
+        "usage": {
+          "type": "recharge on dice",
+          "dice": "1d6",
+          "values": "5-6"
```

Comment:
Would making this an array of integers make more sense?

Reply:
Yeah, reading this isn't immediately obvious to me what it means, its a bit different than other ruling.  Usually you see "3x per day" or something, I forget off the top of my head how conditional recharges work.  I have to look up to have an opinion :) 

But I do like the data extracted like this.

Reply:
Yes, that makes much more sense! I should have done it that way. I'll use the expanded values (`[4, 5, 6]`) instead of a shortcut range notation that would require a full object with boundary properties.

Reply:
Either  way is fine, honestly.